scripts/bump.sh

```diff
@@ -8,8 +8,8 @@ echo "CHANNEL: $CHANNEL"
 PRISMA_VERSION=$2
 echo "PRISMA_VERSION: $PRISMA_VERSION"
 
-# TODO: remove this if-condition once we move to dev
-if [ "$CHANNEL" = "dev" ]; then
```

Comment:
Can we really do this? In other contextes the `channel` was the npm tag I think. Or not?

Reply:
Probably not, as the npm channel currently _is_ still called `alpha`.

Reply:
I think `PRISMA_CHANNEL` is the npm tag, which gets set in the line below.

Reply:
This is indeed the npm channel of the Prisma CLI, it should be kept as is.

Reply:
Can we refactor this to be `NPM_TAG` or `NPM_DIST_TAG` then? That would make it explicit.